codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java

```diff
@@ -341,4 +395,12 @@ public void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
         super.handlerRemoved(ctx);
         cleanup();
     }
+
+    public final int blockSize() {
+        return blockSize;
+    }
+
+    public final int currentBlockLength() {
+        return currentBlockLength;
```

Comment:
UPDATE 2: ok, problem is only with current mock up of removing `currentBlockLength`. Please review away :)